lib/proposed/app.rb

```diff
@@ -0,0 +1,12 @@
+class App
+  def is_human?(name: )
```

Comment:
Cały flow aplikacji, można to jeszcze uprościć, ale i tak od razu widać czytelnie jaka jest logika.